nuxt.config.js

```diff
@@ -27,8 +18,22 @@ export default defineNuxtConfig({
         // Main-Process entry file of the Electron App.
         entry: "electron/main.js",
       },
+      {
+        entry: "electron/preload.js",
+        onstart(args) {
+          args.reload();
+        },
```

Comment:
What is this?

Reply:
https://github.com/caoxiemeihao/nuxt-electron/blob/d7cf10ddaf8e050564d46b0a906cc7a468573e60/quick-start/nuxt.config.ts#L13